### DISCUSSION TITLE
Gagal saat membuat dynamic Endpoint

### DISCUSSION THREAD

**Discussion by isal-amir** [teamalgoritma/community#369] [Q&A] [answered]
Saya mencoba membuat dynamic endpoint namun terdapat error sehingga mendapat warning garis bawah kuning di VScode
![image](https://user-images.githubusercontent.com/79739081/193235460-61698395-e81e-48ec-93e7-e107f44f2d60.png)

dan ketika run dicoba di browser muncul error ini
![image](https://user-images.githubusercontent.com/79739081/193235656-199738e6-400f-4b6e-bea1-d1c88fbdc7f9.png)

**Comment by kevbow** [comment, marked as answer]
Halo @isal-amir, 
bisa coba ditunjukkan untuk kode di vscode-nya?
Terima kasih

**Reply by isal-amir** [reply]
Halo Kak @kevbow 
sudah solved Kak, jadi saya awalnya mengetik manual untuk program ini
![image](https://user-images.githubusercontent.com/79739081/193238790-6ca3dba9-9ac4-4ae1-accf-9643102d656a.png)
namun setelah saya coba dengan copy paste dari notebook, garis kuningnya hilang. sepertinya karena saya ada tambahan tanda / di setelah <station_id> jadi error

**Reply by isal-amir** [reply]
Terima kasih Kak @kevbow
